gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/ApisResource.java

```diff
@@ -261,6 +261,9 @@ public ApiListItemPagedResult getApis(
     @Permissions({ @Permission(value = RolePermission.ENVIRONMENT_API, acls = RolePermissionAction.CREATE) })
     public Response createApi(@Parameter(name = "api", required = true) @Valid @NotNull final NewApiEntity newApiEntity)
         throws ApiAlreadyExistsException {
+        if (DefinitionVersion.valueOfLabel(newApiEntity.getGraviteeDefinitionVersion()).equals(DefinitionVersion.V1)) {
```

Comment:
If `.getGraviteeDefinitionVersion()` returns a value that is not in the enum of DefinitionVersion, there will be a NPE because `DefinitionVersion.valueOfLabel()` will return null. 
It would be safer to switch: `DefinitionVersion.V1.equals(DefinitionVersion.valueOfLabel(newApiEntity.getGraviteeDefinitionVersion())`

Reply:
(Same for the other lines added in this PR)